pkg/vm/opcode/opcode_prices.go

```diff
@@ -0,0 +1,199 @@
+package opcode
+
+import "github.com/nspcc-dev/neo-go/pkg/util"
+
+// GetPrice returns the deployment prices of specified opcodes
+func GetPrice(opcodes ...Opcode) util.Fixed8 {
+	var result util.Fixed8
+	for _, op := range opcodes {
+		result += util.Fixed8(prices[op])
+	}
+	return result
+}
+
+var prices = map[Opcode]int{
+	PUSHINT8:   30,
+	PUSHINT32:  30,
+	PUSHINT64:  30,
+	PUSHINT16:  30,
+	PUSHINT128: 120,
+	PUSHINT256: 120,
+	//PUSHA: 120,
+	PUSHNULL:  30,
+	PUSHDATA1: 180,
+	PUSHDATA2: 13000,
+	PUSHDATA4: 110000,
+	PUSHM1:    30,
+	PUSH0:     30,
+	PUSH1:     30,
+	PUSH2:     30,
+	PUSH3:     30,
+	PUSH4:     30,
+	PUSH5:     30,
+	PUSH6:     30,
+	PUSH7:     30,
+	PUSH8:     30,
+	PUSH9:     30,
+	PUSH10:    30,
+	PUSH11:    30,
+	PUSH12:    30,
+	PUSH13:    30,
+	PUSH14:    30,
+	PUSH15:    30,
+	PUSH16:    30,
+	NOP:       30,
+	JMP:       70,
+	//JMP_L: 70,
+	JMPIF: 70,
+	//JMPIF_L: 70,
+	JMPIFNOT: 70,
+	//JMPIFNOT_L: 70,
+	JMPEQ: 70,
+	//JMPEQ_L: 70,
+	JMPNE: 70,
+	//JMPNE_L: 70,
+	JMPGT: 70,
+	//JMPGT_L: 70,
+	JMPGE: 70,
+	//JMPGE_L: 70,
+	JMPLT: 70,
+	//JMPLT_L: 70,
+	JMPLE: 70,
+	//JMPLE_L: 70,
+	CALL: 22000,
+	//CALL_L: 22000,
+	//CALLA: 22000,
```

Comment:
Why these are commented out?

Reply:
Uncommented now. However, some of them are still not present in neo-go.